docs/concepts/what-is-a-workflow-execution.md

```diff
@@ -2,57 +2,122 @@
-   Start with infinite timeouts.
-3. The SDKs come equipped with timers and sleep APIs that can be used directly inside of Workflows to handle business logic related timeouts.
+- Completed: The Workflow Execution has completed successfully.
+- Continued-As-New: The Workflow Execution [Continued-As-New](/docs/concepts/what-is-continue-as-new)
+- Terminated: The Workflow Execution was terminated.
+- Cancelled: The Workflow Execution successfully handled a cancellation request.
+- Failed: The Workflow Execution returned an error and failed.
+- Timed Out: The Workflow Execution reached a timeout limit.
 
-Workflows intended to run indefinitely should be written with some care.
-Temporal stores the complete event history for the entire lifecycle of a Workflow Execution.
-There is a maximum limit of 50,000 events that is enforced by the Server, and you should try to avoid getting close to this limit; The Temporal Server puts out a warning at every 10,000 events.
+### Workflow Execution Chain
 
-The idiomatic way to handle indefinitely running Workflows is to use the "Continue-as-new" feature, which is available in all SDKs.
-For example, a reasonable cutoff point might be once a day for high volume Workflows.
+A Workflow Execution Chain is a sequence of Workflow Executions that share the same Workflow Id.
+Each link in the Chain is often called a Workflow Run.
+Each Workflow Run in the sequence is connected by one of the following:
 
-The "Continue-as-new" feature completes the current Workflow execution and automatically starts a new execution with the same Workflow Id, but different run Id, passing it the appropriate parameters for it to continue.
-This keeps the event history within limits, but continues the logic execution.
+- [Temporal Cron Job](/docs/concepts/what-is-a-temporal-cron-job)
+- [Continue As New](/docs/concepts/what-is-continue-as-new)
+* [Retries](/docs/concepts/what-is-a-retry-policy)
 
-**How can I load test Workflow Executions?**
+A Workflow Execution is uniquely identified by its [Namespace](/docs/concepts/what-is-a-namespace), [Workflow Id](/docs/concepts/what-is-a-workflow-id), and [Run Id](/docs/concepts/what-is-a-run-id).
 
-The [Temporal stress testing blog post](https://docs.temporal.io/blog/temporal-deep-dive-stress-testing) covers many different scenarios under which we test Workflow Executions.
+The [Workflow Execution Timeout](/docs/concepts/what-is-a-workflow-execution-timeout) applies to a Workflow Execution Chain.
+The [Workflow Run Timeout](/docs/concepts/what-is-a-workflow-run-timeout) applies to a single Workflow Execution (Workflow Run).
 
-**Implementation guides:**
+### Event loop
 
-- [How to spawn a Workflow Execution in Go](/docs/go/how-to-spawn-a-workflow-execution-in-go)
+A Workflow Execution is made up of a set of [Events](/docs/concepts/what-is-an-event) called an [Event History](/docs/concepts/what-is-an-event-history).
```

Comment:
"...set of events..." - would calling this a '...sequence of events...' help contextualize it better as a log-based thing?